debian/changelog

```diff
@@ -1,4 +1,4 @@
-forstfs-s3-gw (0.0.0) stable; urgency=medium
+frostfs-s3-gw (0.0.0) stable; urgency=medium
```

Comment:
Do we need this file to be committed if we added it to `.gitignore`?

Reply:
This file need for locally build debian packages